api/specs/web-server/_storage.py

```diff
@@ -228,11 +228,11 @@ async def is_completed_upload_file(
 
 
 @router.post(
-    "/storage/locations/{location_id}/export-data",
+    "/storage/locations/{location_id}/data-export",
     response_model=Envelope[TaskGet],
-    name="export_data",
+    name="data_export",
     description="Export data",
     responses=_data_export_responses,
 )
-async def export_data(data_export: DataExportPost, location_id: LocationID):
+async def data_export(data_export: DataExportPost, location_id: LocationID):
```

Comment:
minor: this naming does not make sense. the action is to export. not sure I get why `export_data` is changed...

Reply:
correct version rename everywhere `export_data`